src/NHibernate.Test/Linq/QueryLock.cs

```diff
@@ -17,6 +19,11 @@ protected override bool AppliesTo(Dialect.Dialect dialect)
 			return TestDialect.SupportsSelectForUpdate;
 		}
 
+		protected override bool AppliesTo(ISessionFactoryImplementor factory)
+		{
+			return factory.ConnectionProvider.Driver is OdbcDriver;
```

Comment:
Only ODBC supports this? I guess a `!` is missing.

Reply:
Yes, hahah